src/validation.ts

```diff
@@ -23,6 +29,65 @@ export function assertAndGetScraperEnvVars() {
     scraperApiJwt: process.env.SCRAPER_API_JWT
   };
 }
+
+export function assertAndGetOwnerWallet() {
+  assert(process.env.INFURA_API_KEY, "Env var INFURA_API_KEY missing");
```

Comment:
Similar to above, prefer generalized node urls over api keys for a specific provider.